worlds/smz3/Options.py

```diff
@@ -25,7 +25,7 @@ class SwordLocation(Choice):
     Randomized - The sword can be placed anywhere.
     Early - The sword will be placed in a location accessible from the start of
     the game.
-    Unce assured - The sword will always be placed on Link's Uncle."""
+    Uncle - The sword will always be placed on Link's Uncle."""
```

Comment:
Should this be `Uncle Assured`?

Reply:
The option itself is "Uncle", makes more sense to match. Probably fine either way.